managing_providers/_topics/embedded_workflows.md

```diff
@@ -460,3 +460,12 @@ You can create a generic service catalog item that uses an embedded workflow. To
    The list of services and requests is shown when the catalog item is submitted. Clicking the request shows the execution status, including any embedded workflows.
 
    ![Workflow Status](../images/embedworkflow_runstatus.png)
+
+### Floe Features Not Yet Supported
+
+The following are not yet supported for Floe:
+- Parallel State
+- Map State Fields:
+  - ItemReader
+  - ItemSelector/ItemBatcher
+  - ResultWriter
```

Comment:
I see on line 167 we say we don't support Map and Parallel, can you update this to say we don't support these four Map state fields and we can merge this once we pull in a new release of floe?